.pylintrc

```diff
@@ -61,6 +61,9 @@ confidence=
 # no Warning level messages displayed, use "--disable=all --enable=classes
 # --disable=W".
 disable=print-statement,
+        line-too-long,
+        bad-continuation,
+        missing-function-docstring,
```

Comment:
From what I understand `line-too-long` and `bad-continuation` will continue to be checked in the project. Given that this is under the `[MESSAGES CONTROL]` [messaging control in pylint](http://pylint.pycqa.org/en/latest/user_guide/message-control.html) seems like then flag the properties that sometimes we want to enable/disable

Example, if I don't add the lines here in this section
```py
long_line = "abcdfgh" # pylint disable=line-too-long
```
Pylint would still lint the `line-too-long` rule.

Reply:
This will disable the rules for the entire project, without the need of a `pylint: disable`.

The comment `pylint: disable` itself should work without anything else, for example we use [`# pylint: disable=broad-except`](https://github.com/ChauffeurPrive/nestor-api/blob/09284992c5552a007d84bca95b6010a0bca69c8c/nestor_api/lib/git.py#L101) which is not in the `.pylintrc` file

Reply:
I don't really have a better approach, maybe use Pycharm Code > reformat code tool, it usually handles properly long lines and indenting

Reply:
Solved 🙏  thanks !